lib/utils/util.js

```diff
@@ -0,0 +1,59 @@
+/** @flow */
+
+import {spawn} from 'child_process';
+import {window, workspace} from 'vscode'
+
+const NODE_NOT_FOUND = '[Flow] Cannot find node in PATH. The simpliest way to resolve it is install node globally'
+const FLOW_NOT_FOUND = '[Flow] Cannot find flow in PATH. Try to install it by npm install flow-bin -g'
+
+function getPathToFlow(): string {
+	return workspace.getConfiguration('flow').get('pathToFlow')
+}
+
+export function isFlowEnabled() {
+	return workspace.getConfiguration('flow').get('enabled')
+}
+
+export function checkNode() {
```

Comment:
I'm still hesitant about this. What if you have Flow installed by other means, and you are using `nvm` to manage your local project, and you have no global node. There is no reason you need it. However, this will still pop up every time you start your editor.

Would you be willing to compromise by doing the following?

* Include a note in the message that it only applies if you've installed Flow with `flow-bin`
* Add a setting to disable the popup and reference it in the error message.